main.js

```diff
@@ -1,10 +1,28 @@
 var $front = document.querySelector('#front-img')
 var $back = document.querySelector('#back-img')
-var $optionsForm = document.querySelector('#options-form')
+var $genderForm = document.querySelector('#gender')
+var $primaryColor = document.querySelector('#jersey-color-primary')
+var $secondaryColor = document.querySelector('#jersey-color-secondary')
+var primaryColor = 'white'
+var secondaryColor = 'white'
+var gender = 'male'
```

Comment:
instead of declaring these globally, they should probably be inputs to the `changeJersey` function. You may also want to keep a "source of truth" object describing the jersey configuration:

```js
var jerseyConfig = {
  gender: '...',
  primaryColor: '...',
 ...
}
```